### ISSUE TITLE
Submodules should use https instead of ssh

### ISSUE THREAD

**Issue by IceReaper** [xamarin/urho#372]
Cloning the submodules only works if you have a local ssh key which is added to your github account. Otherwise the submodules cannot be cloned due to an access denied error. As these repositories are public, using https sources will fix this problem.

**Comment by Therzok** [comment]
`url = ../../owner/repo.git` will use the same protocol as the main clone.

**Comment by utekai** [comment]
Yes, I complained about that at some point a long, long time ago ... but then installed a local ssh key. 

@Therzok  Can you show the full command?

Here's the command I used.
https://github.com/xamarin/urho.git  --recursive
